backend/src/main/java/com/capstone/moneytree/model/decorator/OrderDecorator.java

```diff
@@ -0,0 +1,15 @@
+package com.capstone.moneytree.model.decorator;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.NoArgsConstructor;
+import net.jacobpeterson.domain.alpaca.order.Order;
+
+@Builder
```

Comment:
Remove the @Builder and @NoArgsConstructor here as they are causing the error on the CI pipeline.

Reply:
Done!